### DISCUSSION TITLE
Add the ability to add users in the client

### DISCUSSION THREAD

**Discussion by 19peytonsmith** [ppy/osu#25918] [Ideas]
When I click on a user's profile I expect (like in the web) to be able to add them as a friend like so:
![image](https://github.com/ppy/osu/assets/54728307/6ab0da3f-2406-4bd6-ab97-94b1abe55c4d)

Not sure if there is an ability to do this already (if so the UI is unintuitive at first glance and should be corrected). Another option would be add an option here, that would make sense:
![image](https://github.com/ppy/osu/assets/54728307/6f8c3ede-6f8a-4060-9b00-08641f38d60a)

**Comment by bdach** [comment]
Yeah we don't have the means to add this right now (https://github.com/ppy/osu-web/issues/10789)
